.travis.yml

```diff
@@ -73,22 +76,26 @@ matrix:
           env: PYTHON_VERSION=3.5 NUMPY_VERSION=1.10 SETUP_CMD='test --open-files'
 
         # Now try with all optional dependencies the latest 3.x and on 2.7.
-        # (with latest numpy).
+        # (with latest numpy). We also test the two latest matplotlib versions
+        # for the image tests.
         - os: linux
-          env: PYTHON_VERSION=2.7 SETUP_CMD='test'
+          env: PYTHON_VERSION=2.7 SETUP_CMD='test -a "--mpl"'
                CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
-               if [[ $PYTHON_VERSION == 2.7 ]]; 
-               then PIP_DEPENDENCIES='jplephem mock';
-               else PIP_DEPENDENCIES='jplephem'; 
+               if [[ $PYTHON_VERSION == 2.7 ]];
```

Comment:
I still think we need to get rid of this if/else. It's unrelated to this PR though, so if you prefer I'm happy to open a separate one.